.github/workflows/release.yml

```
@@ -0,0 +1,217 @@
+# Copyright (c) 2010-2022, Lawrence Livermore National Security, LLC. Produced
+# at the Lawrence Livermore National Laboratory. All Rights reserved. See files
+# LICENSE and NOTICE for details. LLNL-CODE-443271.
+#
+# This file is part of the GLVis visualization tool and library. For more
+# information and source code availability see https://glvis.org.
+#
+# GLVis is free software; you can redistribute it and/or modify it under the
+# terms of the BSD-3 license. We welcome feedback and contributions, see file
+# CONTRIBUTING.md for details.
+
+# GLVis - an OpenGL visualization server based on the MFEM library
+
+name: release
+
+on:
+  push:
+    tags:
+      - v*
+    branches:
+      - master
```

Any objections to add manual triger?

```suggestion
on:
  push:
    tags:
      - v*
    branches:
      - master
  workflow_dispatch:
```

---

This will make sense also in `builds.yml` for branches without PR.

We added it recently in mfem, see https://github.com/mfem/mfem/actions/workflows/repo-check.yml